roles/etcd/defaults/main.yaml

```diff
@@ -17,7 +19,12 @@ etcd_image: "{{ etcd_image_dict[openshift_deployment_type | default('origin')] }
 # etcd run on a host => use etcdctl command directly
 # etcd run as a docker container => use docker exec
 # etcd run as a runc container => use runc exec
-r_etcd_common_etcdctl_command: "{{ 'etcdctl' if (r_etcd_common_etcd_runtime == 'host') else ('docker exec etcd_container etcdctl' if (r_etcd_common_etcd_runtime == 'docker') else 'runc exec etcd etcdctl') }}"
+etcdctl_dict:
```

Comment:
While technically correct, I liked the previous implementation due to the fact that we were getting rid of half of it in 3.11.  This way is less clear, it's not clear what's doing what or why.

Reply:
Right, I'm not entirely sure which options should be removed - so I kept those. FAH tests would verify only minor update of static pods case